aws-common/src/main/java/io/druid/common/aws/AWSProxyConfig.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.common.aws;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AWSProxyConfig
+{
+  @JsonProperty
+  private String host;
+
+  @JsonProperty
+  private int port = -1;
+
+  @JsonProperty
+  private String username;
+
+  @JsonProperty
+  private String password;
+
+  @JsonProperty
+  public String getHost()
+  {
+    return host;
+  }
+
+  @JsonProperty
+  public int getPort()
+  {
+    return port;
+  }
+
+  @JsonProperty
+  public String getUsername()
+  {
+    return username;
+  }
+
+  @JsonProperty
+  public String getPassword()
```

Comment:
can this be a `PasswordProvider`?

Reply:
What do you mean? If you mean `AWSCredentialsProvider`, it can be used here because this is for proxy configuration. If you mean we need a sort of credentialsProvider for proxy, I agree but want to do in a follow-up pr.

Reply:
I mean a http://druid.io/docs/latest/operations/password-provider.html